pkg/timer/timerSync.go

```diff
@@ -98,7 +98,11 @@ func (ws *TimerSync) TimeTriggerEventHandlers(ctx context.Context) {
 			},
 			UpdateFunc: func(_ interface{}, obj interface{}) {
```

Comment:
Instead of checking from map, You can do something like this.

UpdateFunc: func(oldObj interface{}, newObj interface{}) {
					oldTimeTrigger := oldObj.(*fv1.TimeTrigger)
					newTimeTrigger := newObj.(*fv1.TimeTrigger)
					if oldTimeTrigger.ObjectMeta.ResourceVersion != newTimeTrigger.ObjectMeta.ResourceVersion  {
						ws.AddUpdateTimeTrigger(newTimeTrigger)
					}
			}